### ISSUE TITLE
As Portal User, I need to see list of top support organizations in my community so that I can quickly find help I need. 

### ISSUE THREAD

**Issue by aroubin** [LSCOIT/LegalNavigator#64]
_From @czdiana2321 on May 21, 2018 19:39_

USER->UI

Acceptance Criteria:
1. The system is able to return list of top support organization within the user's community.
2. Portal User is able to see list of top support organizations within the user's community.
3. Portal User is able to filter the list based on the location (State/City/Zip).

_Copied from original issue: Microsoft/Access2Justice#81_

**Comment by yfzwinnie** [comment]
Completed as part of MVP.

**Comment by breeglej** [comment]
@KellyeDC @grawdon - Where are the support organizations supposed to appear?  What are we testing here?

**Comment by KellyeDC-zz** [comment]
@breeglej Currently, I think the organizations might not be lists in the most meaningful way to the user.  I believe it is alphabetically for each area (zip code). @svasylenko Please confirm.

**Comment by svasylenko** [comment]
@KellyeDC yes, you are right, it is ordered alphabetically. As for meaningful ordering - please suggest what way we should order them.
